treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -18,9 +18,10 @@
 
       On the page above you can see an alert for each affected platform as well as a link to a graph showing the history of scores for this test. There is also a link to a treeherder page showing the Talos jobs in a pushlog format.
 
-      To learn more about the regressing test(s), please see: https://wiki.mozilla.org/TestEngineering/Performance/Talos
-
-      For information on reproducing and debugging the regression, either on try or locally, see: https://wiki.mozilla.org/TestEngineering/Performance/Talos/Running
+      To learn more about:
+          1. sheriffs' context, please see: https://wiki.mozilla.org/TestEngineering/Performance/Sheriffing/Workflow#Context
```

Comment:
I don't think this should be the first thing in the list. The most useful link would be something that explains what the test is doing. The closest thing we have to that right now is https://wiki.mozilla.org/TestEngineering/Performance/Talos but ultimately we will be able to report to the generated docs. The next most useful thing would be a guide on running the tests, pushing to try, and comparing results. I would say that the sheriffing workflow is useful, as is the FAQ, but these are supplementary. Below I've taken an existing bug comment and updated it to use Markdown and include links to tests. I don't think we can do this yet, but it would be worth moving in this direction. Rather than give lots of links, we should point engineers to a guide that leads them through the regression handling process.

---

Perfherder has detected a performance regression from push [4f83b17d63bffb866843ca3a687b33d94a8b84cd](https://hg.mozilla.org/integration/autoland/pushloghtml?changeset=4f83b17d63bffb866843ca3a687b33d94a8b84cd). As author of one of the patches included in that push, we need your help to address this regression.

**Regressions**
* **3%** [ts_paint](https://wiki.mozilla.org/TestEngineering/Performance/Talos/Tests#ts_paint) windows10-64-shippable opt e10s stylo 328.88 -> 339.58
* **3%** [startup_about_home_paint](https://wiki.mozilla.org/TestEngineering/Performance/Talos/Tests#startup_about_home_paint) linux64-shippable opt e10s stylo 782.71 -> 803.33
* **3%** [startup_about_home_paint](https://wiki.mozilla.org/TestEngineering/Performance/Talos/Tests#startup_about_home_paint) linux64-shippable opt e10s stylo 783.38 -> 804.00
* **2%** [sessionrestore](https://wiki.mozilla.org/TestEngineering/Performance/Talos/Tests#sessionrestore) linux64-shippable opt e10s stylo 563.38 -> 574.83

Details of the alert can be found in the [alert summary](https://treeherder.mozilla.org/perf.html#/alerts?id=24843), including links to graph and comparisongs for each of the affected tests. Please follow our [guide to handling regression bugs](https://wiki.mozilla.org/TestEngineering/Performance/Talos/RegressionBugsHandling) and **let us know your plans within 3 business days, or the offending patch(es) will be backed out** in accordance with our [regression policy](https://www.mozilla.org/en-US/about/governance/policies/regressions/).

For more information on performance sheriffing please see our [FAQ](https://wiki.mozilla.org/TestEngineering/Performance/FAQ).